TI-reports/2025/2025-Q1-Alpha-Omega.md

```diff
@@ -0,0 +1,58 @@
+# 2025 Q1 Alpha-Omega
+
+## Overview
+
+Alpha-Omega enters 2025 with renewed funding from Google and Amazon, a healthy pipeline of potential engagements, and updated OKRs to guide and focus on our investments. We're thankful to have such an engaging security community around us and we're looking forward to seeing their accomplishments this year.
+
+## Recent Events / News / Blogs / Etc.
+
+* [What's in the SOSS? Podcast #21 – Alpha-Omega's Michael Winser and Catalyzing Sustainable Improvements in Open Source Security](https://openssf.org/podcast/2024/12/10/whats-in-the-soss-podcast-21-alpha-omegas-michael-winser-and-catalyzing-sustainable-improvements-in-open-source-security/) (CRob, Michael Winser [Alpha-Omega])
+* FOSDEM: [How FreeBSD security audits have improved our security culture](https://fosdem.org/2025/schedule/event/fosdem-2025-6152-how-freebsd-security-audits-have-improved-our-security-culture/) (Pierre Pronchery [FreeBSD], Michael Winser [Alpha-Omega])
+* FOSDEM: [Funding FOSS together: Combining public and private efforts](https://fosdem.org/2025/schedule/event/fosdem-2025-5279-funding-foss-together-combining-public-and-private-efforts/) (Mirko Swillus [Sovereign Tech Fund], Michael Winser [Alpha-Omega])
+* FOSDEM: [Airflow Beach Cleaning - Securing Supply Chain](https://fosdem.org/2025/schedule/event/fosdem-2025-4594-airflow-beach-cleaning-securing-supply-chain/) (Jarek Potiuk [Apache], Munawar Hafiz [OpenRefactory], Michael Winser [Alpha-Omega])
+* [An Overview of Cyber Security Funding for Open Source Software](https://arxiv.org/pdf/2412.05887) (Jukka Ruohonen, Gaurav Choudharya, Adam Alami [University of Southern Denmark]
+
+
+## Upcoming Events
+
+* VulnCon: **Airflow Beach Cleaning - Supply Chain Security with Community in Mind** (Jarek Potiuk [Apache], Michael Winser [Alpha-Omega])
+* VulnCon: **Alpha-Omega: What We've Learned From Funding Open Source Security Over the Past 3 Years, What's Ahead** (Michael Winser [Alpha-Omega])
+* Alpha-Omega Public Meeting - April 2, 2025 -- **please join us!**
+
+## Objectives & Key Results
+
+#### Objective #1: Catalyze trustworthy and secure software, runtimes, and infrastructure for all the major open source ecosystems through staffing**
+* **KR1.1**: Fund security improvements and initiatives for at least ten critical open source organizations by the end of 2025.
+* **KR 1.2**: For each engagement, confirm progress toward improved security outcomes, evidenced through initial and/or follow-on assessments, monthly reporting, and periodic check-ins.
+* **KR 1.3**: Drive the organizations we work with to obtain security funding from at least one organization other than Alpha-Omega, targeting 33% by the end of 2025.
+* **KR 1.4**: Organize quarterly roundtables for at least 5 major ecosystems  to share information, build connections, and collaborate, resulting in at least one new project or joint publication started in 2025.
```

Comment:
There's interest in #wg-securing-software-repos about having a PackagingCon-like event in 2025 - could be a good opportunity to collaborate!